ckanext/dcat/profiles.py

```diff
@@ -104,17 +104,47 @@ def _object(self, subject, predicate):
             return _object
         return None
 
-    def _object_value(self, subject, predicate):
+    def _object_value(self, subject, predicate, multilang=False):
         '''
         Given a subject and a predicate, returns the value of the object
 
         Both subject and predicate must be rdflib URIRef or BNode objects
 
         If found, the unicode representation is returned, else an empty string
         '''
+        default_lang = config.get('ckan.locale_default', 'en')
+        lang_dict = {}
         for o in self.g.objects(subject, predicate):
-            return unicode(o)
-        return ''
+            if multilang and o.language:
+                lang_dict[o.language] = unicode(o)
+            elif multilang:
+                lang_dict[default_lang] = unicode(o)
+            else:
+                return unicode(o)
+        if multilang:
+            # when translation does not exist, create an empty one
+            for lang in get_langs():
+                if lang not in lang_dict:
+                    lang_dict[lang] = ''
+        return lang_dict
```

Comment:
To keep the default behavior when nothing is found I suggest moving `return lang_dict` under `if multilang: ` and inserting `return ''` as default return value if `multilang=False` and nothing was found.

Reply:
I agree, that makes sense. I updated my PR accordingly to that.

Reply:
Thank's for your update!